dll/shellext/sendmail/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+set_cpp(WITH_RUNTIME)
+spec2def(sendmail.dll sendmail.spec ADD_IMPORTLIB)
```

Comment:
The `ADD_IMPORTLIB` is only required when someone wants to import `sendmail`

Reply:
Deleted.